pronotepy/dataClasses.py

```diff
@@ -494,6 +498,7 @@ def __init__(self, client: _ClientBase, json_dict: dict) -> None:
 
         self.title: str = self._resolver(str, 'L', strict=False)
         self.description: str = self._resolver(Util.html_parse, "descriptif", "V", strict=False)
+        self.category: str = self._resolver(str, "categorie", "V", "L", strict=False)
```

Comment:
This should be marked Optional since it's not strict. (didn't spot it in the last pr) It should also be changed in the docstring.
```suggestion
        self.title: Optional[str] = self._resolver(str, 'L', strict=False)
        self.description: Optional[str] = self._resolver(Util.html_parse, "descriptif", "V", strict=False)
        self.category: Optional[str] = self._resolver(str, "categorie", "V", "L", strict=False)
```

Reply:
Yes you right Excuse me I don't think about this 😅

Reply:
How work the docstring? How can I update it?

Reply:
[here (dataClasses.py line 484)](https://github.com/bain3/pronotepy/pull/98/files/bb3564151bf13e529a271a7cbfab73d302f8f663#diff-e7e6c320f3108aeadcf3a06df8d80b4d099cac3f27292ade09577bf743f6f018R484) you just change to Optional[str]

Reply:
Yes I will do that this evening